tasks.py

```diff
@@ -187,6 +241,17 @@ def deploy(ctx, space=None, branch=None, login=False, help=False):
     with open(".cfmeta", "w") as fp:
         json.dump({"user": os.getenv("USER"), "branch": branch}, fp)
 
+    # Runs migrations
+    # tasks.py does not continue until the migrations task has completed
+    migrator_app = 'fecfile-api-migrator'  # the app with this name will get deleted!
```

Comment:
Can we make this a constant like `WEB_SERVICES_NAME`?

Reply:
Done :+1: